vpr/src/route/router_lookahead_map_utils.cpp

```diff
@@ -288,15 +285,13 @@ void expand_dijkstra_neighbours(const t_rr_graph_storage& rr_nodes,
     }
 }
 
-template void expand_dijkstra_neighbours(const t_rr_graph_storage& rr_nodes,
-                                         const PQ_Entry_Delay& parent_entry,
+template void expand_dijkstra_neighbours(const PQ_Entry_Delay& parent_entry,
```

Comment:
We should avoid the use of global variables.
Can you just replace the ``const t_rr_graph_storage& rr_nodes,`` with ``const RRGraphView& rr_graph``?

Reply:
We have already removed rr_nodes from here.  Because that was unused. And if it is used in some other files then edge_range API should be in rr_graph to remove rr_node from those files, as I mentioned above in comment.

Reply:
Removing the rr_nodes is correct. Can you bring the RRGraphView to the input argument list of this function?

For edge_range, I will read more codes and tell you how to proceed. For this PR, it seems o.k. We will rework it in later PRs. Let me know if you have any question.

Reply:
OK that is fine. Today in morning I will do that. I have already run the QoR tests should I run again after putting this argument.

Reply:
Yes. We need to rerun the regression because this function is called many times in aglorithms.
Also, feel free share existing QoR test results. I can take a look.

Reply:
These are results before adding RRGraphView& rr_graph in arguments.
[titan_quick_qor.xlsx](https://github.com/verilog-to-routing/vtr-verilog-to-routing/files/7750206/titan_quick_qor.xlsx)
[vtr_reg_qor_chain_depop.xlsx](https://github.com/verilog-to-routing/vtr-verilog-to-routing/files/7750207/vtr_reg_qor_chain_depop.xlsx)

Reply:
The QoR result looks good. is it against current master or a specific branch ``test_basic_sanity``?
I think the final QoR should also look fine.

Reply:
Its against test_basic_sanity.